internal/rpc/client_test.go

```diff
@@ -507,6 +508,27 @@ func newServer(f func(*websocket.Conn) error) *server {
 	return &srv
 }
 
+func newIPv6Server(f func(*websocket.Conn) error) *server {
+	var srv server
+	l, _ := net.Listen("tcp", "[::1]:0")
+	server := httptest.Server{
+		Listener: l,
+		Config:   &http.Server{Handler: handleWS(f)}, //nolint:gosec
+	}
+	server.StartTLS()
+	srv.Server = &server
+	srv.URL = "ws" + strings.TrimPrefix(srv.Server.URL, "http")
```

Comment:
wss?

Reply:
`<http>s://google.com` -> replace `http` with `ws` -> `https://google.com`

this code works both for HTTP and https url